js/main.js

```diff
@@ -215,35 +214,54 @@ var mobileNav = function mobileNav() {
 /*!*************************************************!*\
   !*** ./resources/js/modules/show-hide/index.js ***!
   \*************************************************/
-/*! no static exports found */
-/***/ (function(module, exports) {
+/*! exports provided: default */
+/***/ (function(module, __webpack_exports__, __webpack_require__) {
+
+"use strict";
+__webpack_require__.r(__webpack_exports__);
+/* harmony import */ var _utilities_selectors__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! @utilities/selectors */ "./resources/js/utilities/selectors/index.js");
+
+
+function loopThroughChildren(parent, className) {
```

Comment:
Is this just a helper function whilst you are writing it to log out errors? It doesn't seem to have a purpose in the final piece of code?